pages/map.js

```diff
@@ -138,7 +138,9 @@ export default function Map({ data }) {
                 />
               ))}
         </div>
-        <DynamicMap users={filteredUsers.length > 0 ? filteredUsers : users} />
+        <div style={{height: '100vh'}}>
```

Comment:
I think we can use Tailwind here `h-screen` which is `height: 100vh;` https://tailwindcss.com/docs/height